Tests/Creation/Frame/Content/String/ID3CreditsListFrameCreatorTest.swift

```diff
@@ -0,0 +1,58 @@
+//
+//  ID3CreditsListFrameCreatorTest.swift
+//  ID3TagEditor
+//
+//  Created by Nolaine Crusher on 4/2/20.
+//  Copyright © 2020 Fabrizio Duroni. All rights reserved.
+//
+
+import XCTest
+@testable import ID3TagEditor
+
+class ID3CreditsListFrameCreatorTest: XCTestCase {
+    func testCreatorFrameCreditsList() {
+        let id3CreditsListFrameCreator = ID3CreditsListFrameCreator(
+            frameContentSizeCalculator: ID3FrameContentSizeCalculator(
+                uInt32ToByteArrayAdapter: UInt32ToByteArrayAdapterUsingUnsafePointer(),
+                synchsafeEncoder: SynchsafeIntegerEncoder()
+            ),
+            frameFlagsCreator: ID3FrameFlagsCreator(),
+            stringToBytesAdapter: ID3UTF16StringToByteAdapter(
+                paddingAdder: PaddingAdderToEndOfContentUsingNullChar(),
+                frameConfiguration: ID3FrameConfiguration()
+            )
+        )
+        
+        let frameBytes = id3CreditsListFrameCreator.createFrame(
+            frameIdentifier: [0x22],
+            version: .version3,
+            role: "musician",
+            person: "Musician Name")
+        
+        XCTAssertEqual(
+            frameBytes,
+            [
+                0x22, // Identifier (See above.)
+                0x00, 0x00, 0x00, 0x34, // Size
+                0x00, 0x00, // Flags
+                0x01, // UCS‐2
+                0x6d, 0x75, 0x73, 0x69, 0x63, 0x69, 0x61, 0x6e, 0x00, // “musician”
+                0x4d, 0x75, 0x73, 0x69, 0x63, 0x69, 0x61, 0x6e, 0x20, 0x4e, 0x61, 0x6d, 0x65, 0x00,// “Musician Name”
+            ]
+        )
+    }
+}
+
+/* XCTAssertEqual failed: ("[
```

Comment:
This test is coming out all sorts of wonky. It seems to be expecting a null byte between each character?

Reply:
You’ve hooked it to a `ID3UTF16StringToByteAdapter`. UTF‐16 uses two bytes (16 bits) for each character. In the ASCII range, the first half is a zero byte and the second half is the familiar byte.